### DISCUSSION TITLE
Show recipes that are valid for this machine

### DISCUSSION THREAD

**Discussion by imcasper** [TechReborn/TechReborn#2550] [Ideas]
In the GUI of the machine, in some mods, by clicking on the arrow, the actual recipes were displayed.
It is very convenient.

Probably it is necessary to contact the help-mod, but I'm not sure, so I am writing here.

**Comment by xanthian** [comment]
I think the Forge version had this, or at least an older version 
![image](https://user-images.githubusercontent.com/7688001/140800561-5e7386a3-e7a5-435f-b10e-f3ae4ac10594.png)

(JEI button in top left)
I did mention it to shedaniel of REI and he suggested that to 
"mention it to the mod developers that there is a builtin way in rei to set bounds of the arrow widget of their screens, much like how in rei you can click the arrow in crafting tables and such"
:)

**Comment by Ayutac** [comment]
one could also add a recipe book as with vanilla probably
